gcsfilesystem/src/main/scala/cromwell/gcsfilesystem/GoogleConfigurationAdapter.scala

```diff
@@ -1,34 +1,26 @@
-package cromwell.engine.io.gcs
+package cromwell.gcsfilesystem
 
 import com.typesafe.config.{Config, ConfigFactory}
-import cromwell.util.ConfigUtil._
+import lenthall.config.ValidatedConfig._
 
 import scala.util.Try
 import scalaz.Scalaz._
 import scalaz._
 
-// Google Authentication modes supported for Cromwell
-sealed trait GoogleCromwellAuthMode
-final case class ServiceAccountMode(accountId: String, pemPath: String) extends GoogleCromwellAuthMode
-final case class UserMode(user: String, secretsFile: String, datastoreDir: String) extends GoogleCromwellAuthMode
-case object ApplicationDefaultMode extends GoogleCromwellAuthMode
-
-// Google Authentication modes supported for User
-sealed trait GoogleUserAuthMode
-final case class Refresh(clientSecrets: ClientSecrets) extends GoogleUserAuthMode
+final case class GoogleConfigurationAdapter(appName: String, cromwellAuthMode: GoogleAuthMode, userAuthMode: Option[GoogleAuthMode]) {
```

Comment:
This class is the old `GoogleConfiguration` and is there temporarily as a compatibility workaround, knowing that the config template is going to change soon anyway.